README.md

```diff
@@ -42,6 +42,7 @@ node ./bin/gltf-pipeline.js -i ./specs/data/boxTexturedUnoptimized/CesiumTexture
 |`--removeNormals`, `-r`|Strips off existing normals, allowing them to be regenerated.|No, default `false`|
 |`--faceNormals`, `-f`|If normals are missing, they should be generated using the face normal.|No, default `false`|
 |`--tangentsBitangents`|If normals and texture coordinates are given, generate tangents and bitangents.|No, default `false`|
+|`--stats`|Generate statistics for input and output glTF files.|No, default `false`|
```

Comment:
Mention here and in `parseArguments` that this prints the statistics to the console.